Out of memory error on the last epoch :/ it's fine, it wasn't good anyway, no corresponding model for this one

hyperparams:
context_length = 512
    d_model = 512
    num_heads = 8
    num_hidden_layers = 4
    d_hidden = 2048
    num_encoders = 8
    num_epochs = 15
    mod_name = 'mod_2'



TLDR: steady decrease in accuracy and 
EPOCH 6
'f1_train': [0.8333333333333334, 0.6666666666666666, 0.3333333333333333, 0.3333333333333333, 0.8333333333333334, 0.6666666666666666, 0.5], 
'acc_train': [{'accuracy': 0.8044941190239335}, {'accuracy': 0.5287319328222834}, {'accuracy': 0.4712680671777167}, {'accuracy': 0.5287319328222834}, {'accuracy': 0.5287319328222834}, {'accuracy': 0.5287319328222834}, {'accuracy': 0.5287319328222834}], 
'loss_train': [tensor(0.5314, device='cuda:0', grad_fn=<MeanBackward0>), tensor(0.5691, device='cuda:0', grad_fn=<MeanBackward0>), tensor(0.6925, device='cuda:0', grad_fn=<MeanBackward0>), tensor(0.6921, device='cuda:0', grad_fn=<MeanBackward0>), tensor(0.6918, device='cuda:0', grad_fn=<MeanBackward0>), tensor(0.6917, device='cuda:0', grad_fn=<MeanBackward0>), tensor(0.6918, device='cuda:0', grad_fn=<MeanBackward0>)], 
'prec_train': [0.75, 0.6666666666666666, 0.0, 0.3333333333333333, 0.8333333333333334, 0.6666666666666666, 0.5], 
'recall_train': [1.0, 1.0, 0.0, 1.0, 1.0, 1.0, 1.0], 

'f1_val': [0.75, 0.75, 0.5, 0.25, 0.25, 0.5, 1.0], 
'acc_val': [{'accuracy': 0.7737575095576188}, {'accuracy': 0.5287183688330602}, {'accuracy': 0.4712816311669397}, {'accuracy': 0.5287183688330602}, {'accuracy': 0.5287183688330602}, {'accuracy': 0.5287183688330602}, {'accuracy': 0.5287183688330602}], 
'loss_val': [0.4515966291294001, 0.6919869145885479, 0.6937571244295334, 0.6923957885266044, 0.6925154468031083, 0.691884653320132, 0.6914599869709348], 
'prec_val': [0.5, 0.75, 0.0, 0.25, 0.25, 0.5, 1.0], 
'recall_val': [1.0, 1.0, 0.0, 1.0, 1.0, 1.0, 1.0], 


